src/components/profileEdit/ProfileInfo.tsx

```
@@ -116,321 +126,303 @@ export default function ProfileInfo() {
-                                className="hidden"
-                                id="profile-image"
+        <div className="flex flex-col gap-4">
+            <div className="flex gap-5 flex-wrap justify-center">
+                {/* 프로필 이미지 섹션 */}
+                <div className="flex flex-col items-center bg-white rounded-lg shadow py-6 px-10 space-y-4">
+                    <h3 className="text-lg font-medium">프로필 이미지</h3>                            
+                    <div className="relative w-fit">
+                        <button className="bg-red-500 p-1 rounded-full absolute top-0 right-0">
+                            <X className="h-3 w-3 text-white" onClick={handleRemoveImage} />
+                        </button>
+                        <div className="relative w-32 h-32 overflow-hidden rounded-full mb-4">
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)